airflow/sensors/http_sensor.py

```diff
@@ -31,6 +31,15 @@ class HttpSensor(BaseSensorOperator):
     HTTP Error codes other than 404 (like 403) or Connection Refused Error
     would fail the sensor itself directly (no more poking).
 
+    The response check can access the template context by passing ``provide_context=True`` to the operator:
```

Comment:
```suggestion
    The response check can access the template context by passing ``provide_context=True`` to the operator::
```

The double `::` makes the next block a code-block